README.md

```diff
@@ -1,9 +1,29 @@
 # benchHackathonNov20
+This project is focused around developing both an iOS and Android app, using KMM for the shared logic between both platforms.
 
-## Mock-Server
+## The concept
+"CommunityKitchen" is a platform to donate and find excess food for those in need that would otherwise go to waste. 
+Currently, a user can: create an account and login and search for meals in their local area. 
 
-### Installing json-server
-We are currently using `json-server` as a mock server to run the project against.
+## Prerequisites
+
+### Android Studio
+Version 4.2 > is required to compile this project. From the latest changes, Arctic Fox (Canary 2, Beta) is required due to using Compose.
+
+### Xcode
+For running the iOS app via Xcode and for using the simulator, version 12.1 > is required.
+
+### Google Maps Api
+To use the location services and to view the available meals on the map, you will need to add the api key to your project.
+
+### Firebase Console
+We're using Firebse for authentication purposes - for viewing this data and making changes you'll need access to it. 
```

Comment:
Who should I tap up?

Reply:
With the members changing constantly wasn't sure who to put exactly

Reply:
Maybe the slack channel?